e2e-tests/tfm/cypress/e2e/pages/utilisation-reports/utilisationReportPage.js

```diff
@@ -4,10 +4,15 @@ const utilisationReportPage = {
   premiumPaymentsTab: {
     getPaymentLink: (paymentId) => cy.get(`a[data-cy="edit-payment-link--paymentId-${paymentId}"]`),
     clickPaymentLink: (paymentId) => cy.get(`a[data-cy="edit-payment-link--paymentId-${paymentId}"]`).click(),
-    getPremiumPaymentsTableRow: (feeRecordId) => cy.get(`tr[data-cy="premium-payments-table-row--feeRecordId-${feeRecordId}"]`),
     getFacilityIdFilterInput: () => cy.getInputByLabelText('Filter by facility ID'),
     submitFacilityIdFilter: () => cy.get('button[data-cy="facility-filter-submit-button"]').click(),
     generateKeyingDataButton: () => cy.get('[data-cy="generate-keying-data-button"]'),
+    premiumPaymentsTable: {
+      getRow: (feeRecordId) => cy.get(`tr[data-cy="premium-payments-table-row--feeRecordId-${feeRecordId}"]`),
+      getCheckbox: (feeRecordIds, paymentCurrency, status) =>
+        cy.get(`[type="checkbox"][id="feeRecordIds-${feeRecordIds.join(',')}-reportedPaymentsCurrency-${paymentCurrency}-status-${status}"]`),
+      getStatus: (feeRecordId) => cy.get(`[data-cy="premium-payments-table-row--feeRecordId-${feeRecordId}-status"]`),
```

Comment:
Do you think we should remove the `get` prefix? Feels a bit strange as any cypress `.get` is getting something 😄 

Would make consumptions simpler and easier to read IMO, e.g

```js
premiumPaymentsTable.row(id).checkbox()
premiumPaymentsTable.row(id).status()
```